src/tetris/brick/skinStorage.ts

```diff
@@ -0,0 +1,41 @@
+import BrickType from "tetris/brick/brickType";
+import Skin from "tetris/brick/skin";
+import { SkinRarity } from "tetris/brick/skinRarity";
+
+export default class SkinStorage {
+    private skins: Map<BrickType, Array<Skin>>;
+    private equippedSkins: Map<BrickType, Skin>;
+    public skinAmount: number;
+
+    public constructor() {
+        this.skins = new Map<BrickType, Array<Skin>>();
+        this.equippedSkins = new Map<BrickType, Skin>();
+        const skinNameBases = ["colorful", "bw", "sepia", /* rare */ "aqua", "light", "purple", /* epic*/ "flat", "glitch", "shiny", /* legendary*/ "dotted", "relief", "retro",];
+
+        for (let brickType = 0; brickType < 7; brickType++) {
```

Comment:
magic number, you could use `Object.keys(ENUM).length`

Reply:
done